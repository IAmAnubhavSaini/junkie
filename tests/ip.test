#!/bin/sh

NAME="IP"
PCAPS="pcap/gre/fragmented.pcap"
filter() {
	# filter out capture lines since capture payload sizes depends on word size
	grep -v '^Capture' | sed -e 's/^\([^@]\+\)@[0-9a-fx]\+\(.*\)$/\1\2/'
}
OUTPUT="ip.out"

. ./dump_test
