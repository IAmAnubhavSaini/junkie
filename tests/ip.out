-- pcap/gre/fragmented.pcap --
Ethernet: head_len=14, payload=1500, vlan_id=-1, source=00:10:f3:09:19:49, dest=ac:de:48:00:10:99, proto=2048
IPv4: head_len=20, payload=1480, version=4, addr=80.154.67.162->192.168.80.99, proto=GRE, ttl=117

Ethernet: head_len=14, payload=1500, vlan_id=-1, source=00:10:f3:09:19:49, dest=ac:de:48:00:10:99, proto=2048
IPv4: head_len=20, payload=1480, version=4, addr=80.154.67.162->192.168.80.99, proto=GRE, ttl=117

Ethernet: head_len=14, payload=1188, vlan_id=-1, source=00:10:f3:09:19:49, dest=ac:de:48:00:10:99, proto=2048
IPv4: head_len=20, payload=1168, version=4, addr=80.154.67.162->192.168.80.99, proto=GRE, ttl=117
GRE: head_len=4, payload=4124, version=0, protocol=2048, key=None
IPv4: head_len=20, payload=4104, version=4, addr=192.168.208.10->192.168.208.18, proto=ICMP, ttl=128
ICMP: head_len=4104, payload=0, type=EchoReply, err=NONE, id=512

